src/logger/dummy-logger.ts

```diff
@@ -0,0 +1,8 @@
+import  { Logger } from './logger'
+import { LoggerBase } from '@tv2media/logger'
+
+export class DummyLogger extends LoggerBase implements Logger {
```

Comment:
Do we need a dummy logger?
I can see you use it for tests.
Can't you just make a mock of it instead?